src/code/z_demo.c

```diff
@@ -391,176 +448,189 @@ void func_80064824(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
                 }
             }
             break;
-        case 28:
-            play->unk_11DE9 = true;
+
+        case CS_MISC_HALT_ALL_ACTORS:
+            play->haltAllActors = true;
             break;
-        case 29:
-            play->unk_11DE9 = false;
+
+        case CS_MISC_RESUME_ALL_ACTORS:
+            play->haltAllActors = false;
             break;
-        case 30:
-            Flags_SetEnv(play, 3);
+
+        case CS_MISC_SET_FLAG_3:
+            CutsceneFlags_Set(play, 3);
             break;
-        case 31:
-            Flags_SetEnv(play, 4);
+
+        case CS_MISC_SET_FLAG_4:
+            CutsceneFlags_Set(play, 4);
```

Comment:
with other stuff like `CS_MISC_SET_FLAG_LAKE_HYLIA_RESTORED` setting eventchkinf flags, should this be at least for now
```suggestion
        case CS_MISC_SET_CSFLAG_4:
            CutsceneFlags_Set(play, 4);
```